src/components/categories/money/Fortune.vue

```diff
@@ -133,18 +132,28 @@
                     this.symbol = `${this.symbol}/${currency}`;
                     sendMessage(this, "user",
                         `${this.$t('fortune.user.chosenCurrency')} ${currency}`).then(() => {
-                        sendMessage(this, "bot", this.$t('fortune.bot.chooseTime')).then(() => {
-                            this.showTimeButtons = true;
-                        })
+                        fortuneService.getActualDataForSymbol(this.symbol, formatter.formatDate(Date.now())).then(response => {
+                                sendMessage(this, "bot",
```

Comment:
Tutaj obsłużmy też przypadek braku aktualnego kursu. Jeżeli response nie jest success to pokażmy odpowiednią wiadomość